docker-compose/nginx/viewer/app-config.js

```diff
@@ -1,153 +1,49 @@
 window.config = {
-  // default: '/'
-  routerBasename: '/',
-  extensions: ['cornerstone', 'dicom-p10-downloader'],
+  routerBasename: '/ohif-viewer/',
+  modes: [],
+  extensions: [],
   showStudyList: true,
-  filterQueryParam: false,
-  disableServersCache: false,
-  studyPrefetcher: {
-    enabled: true,
-    order: 'closest',
-    displaySetCount: 3,
-    preventCache: false,
-    prefetchDisplaySetsTimeout: 300,
-    displayProgress: true,
-    includeActiveDisplaySet: true,
-  },
-  servers: {
-    dicomWeb: [
-      {
+  // below flag is for performance reasons, but it might not work for all servers
+  showWarningMessageForCrossOrigin: true,
+  strictZSpacingForVolumeViewport: true,
+  showCPUFallbackMessage: true,
+  showLoadingIndicator: true,
+  defaultDataSourceName: 'dicomweb',
+  dataSources: [
+    {
+      namespace: '@ohif/extension-default.dataSourcesModule.dicomweb',
+      sourceName: 'dicomweb',
+      configuration: {
+        friendlyName: 'DCM4CHEE Server',
         name: 'SHANOIR-NG',
-        wadoUriRoot: 'SHANOIR_VIEWER_OHIF_URL_SCHEME://SHANOIR_VIEWER_OHIF_URL_HOST/shanoir-ng/',
-        qidoRoot: 'SHANOIR_VIEWER_OHIF_URL_SCHEME://SHANOIR_VIEWER_OHIF_URL_HOST/dicomweb',
-        wadoRoot: 'SHANOIR_VIEWER_OHIF_URL_SCHEME://SHANOIR_VIEWER_OHIF_URL_HOST/dicomweb',
-        qidoSupportsIncludeField: true,
+        wadoUriRoot: 'SHANOIR_VIEWER_OHIF_URL_SCHEME://SHANOIR_VIEWER_OHIF_URL_HOST/pacs',
```

Comment:
Hi @pierrehenri-dauvergne, thank you for your PR. I think, the URIs should not be changed at all. Your PR does not change anything in the backend, but when you change the URIs how the viewer will get his information from shanoir?